lib/registration.pm

```diff
@@ -198,7 +198,7 @@ sub register_addons_cmd {
     my @addons = grep { defined $_ && $_ } split(/,/, $addonlist);
     foreach my $addon (@addons) {
         my $name = get_addon_fullname($addon);
-        if ($name =~ /module/) {
+        if ($name =~ /adv/ || $name =~ /containers/ || $name =~ /legacy/ || $name =~ /toolchain/ || $name =~ /web-scripting/) {
```

Comment:
this `if` should be 
`if ($name =~ /adv|containers|.../)`

Reply:
Update the code